api/src/main/java/com/eleks/academy/pharmagator/scheduler/Scheduler.java

```diff
@@ -14,19 +20,50 @@
 
 @Slf4j
 @Component
-@Profile("!test")
```

Comment:
Why?
I suppose that all data providers should not run in test contexts.

Reply:
I've just wanted to represent the flow of creating the application. We did not use tests at the time.

Reply:
Try to build it locally! ;)